components/StyledCheckbox.js

```diff
@@ -161,6 +161,7 @@ class StyledCheckbox extends React.Component {
         size={size}
         width={width}
         alignItems={alignItems}
+        name={name}
```

Comment:
Why has this been added?

Reply:
It looks like a leftover from refactor/testing.

Reply:
fixed